Dockerfile.saivs

```diff
@@ -80,8 +80,20 @@ RUN git clone https://github.com/Azure/sonic-sairedis.git \
         && git submodule update --init --recursive \
         && cd .. \
         && ./autogen.sh && ./configure --with-sai=vs && make -j4 \
-        && make install && ldconfig \
-        && cp -R tests .. && rm -rf * && mv ../tests .
+        && make install && ldconfig 
+
+# Build attr_list_generator and generate /etc/sai/sai.json
+COPY scripts/gen_attr_list /sai/gen_attr_list
+
+RUN cd /sai/gen_attr_list \
+        && ./build.sh \
+        && mkdir -p /etc/sai \
+        && ./build/attr_list_generator > /etc/sai/sai.json \
+        && rm -rf /sai/gen_attr_list
+
+# Cleanup sonic-sairedis
```

Comment:
It does not make sense to remove this in the separate Docker layer. The size will remain the same. In case you need some file from the removed files, I'd suggest to copy them somewhere. Also, as an option, sai.json can be generated in the same layer as sonic-sairedis build.